app/components/post_list/post_list.tsx

```diff
@@ -4,7 +4,7 @@
 import {FlatList} from '@stream-io/flat-list-mvcp';
```

Comment:
Just wondering if we can remove this dependency and use regular FlatList, the reason for this dep was to add `maintainVisibleContentPosition` on Android, but it has since then been added to [React Native](https://reactnative.dev/docs/0.77/scrollview#maintainvisiblecontentposition).

Not for this PR, but just something to potentially track with a ticket.

Reply:
Ok please do